src/material/slide-toggle/slide-toggle.scss

```diff
@@ -13,6 +13,11 @@
   // Add the MDC switch static styles.
   @include mdc-switch.static-styles-without-ripple();
 
+  // TODO(wagnermaciel): Use our custom token system to emit this css rule.
+  .mat-mdc-slide-toggle:has(.mdc-switch--disabled) label {
```

Comment:
Couldn't we use the same selector that was in the theme (`.mdc-switch--disabled + label`)? I expect this to make it easier to land since the specificity is lower.

Reply:
Yeah, I'll switch it back